rotkehlchen/data_import/importers/bitstamp.py

```diff
@@ -140,25 +149,31 @@ def _import_csv(self, write_cursor: DBCursor, filepath: Path, **kwargs: Any) ->
         Import trades from bitstamp.
         """
         with open(filepath, encoding='utf-8-sig') as csvfile:
-            data = csv.DictReader(csvfile)
-            for row in data:
+            for index, row in enumerate(csv.DictReader(csvfile), start=1):
                 try:
+                    self.total_entries += 1
                     self._consume_bitstamp_transaction(write_cursor, row, **kwargs)
+                    self.imported_entries += 1
                 except UnknownAsset as e:
-                    self.db.msg_aggregator.add_warning(
-                        f'During Bitstamp CSV import found action with unknown '
-                        f'asset {e.identifier}. Ignoring entry',
+                    self.send_message(
+                        row_index=index,
+                        csv_row=row,
+                        msg=f'Unknown asset {e.identifier}.',
+                        is_error=True,
                     )
                 except DeserializationError as e:
-                    self.db.msg_aggregator.add_warning(
-                        f'Deserialization error during Bitstamp CSV import. '
-                        f'{e!s}. Ignoring entry',
-                    )
-                except KeyError as e:
-                    self.db.msg_aggregator.add_warning(
-                        f'Could not find key {e!s} in csv row {row!s} {e!s}. Ignoring entry',
+                    self.send_message(
+                        row_index=index,
+                        csv_row=row,
+                        msg=f'Deserialization error: {e!s}.',
+                        is_error=True,
                     )
                 except ValueError as e:
-                    self.db.msg_aggregator.add_warning(
-                        f'Could not parse some values {e!s} in csv row {row!s}',
+                    self.send_message(
```

Comment:
here same change to reduce copy pasta. Wont' say it in other places but most probably every importer can improve like this